pkg/network/netlink/conntracker.go

```diff
@@ -67,7 +67,8 @@ type orphanEntry struct {
 	expires time.Time
 }
 
-type realConntracker struct {
+// RealConntracker is the netlink implementation of the Conntracker interface
+type RealConntracker struct {
```

Comment:
I don't think this was meant to be an exported type. I would rather you add a method to the interface that tells you if it is netlink or ebpf.